network/address.go

```diff
@@ -335,6 +335,20 @@ func SelectMongoHostPortsBySpaces(hostPorts []HostPort, spaces []SpaceName) ([]s
 	return HostPortsToStrings(filteredHostPorts), ok
 }
 
+// HostPortWithIPv4Address returns true if the passed slice of HostPort
+// contains an IPv4 address that is not just a loopback address.
+func HostPortWithIPv4Address(hostPorts []HostPort) bool {
```

Comment:
The name sounds like it is returning a HostPort, maybe:

HostPortsHaveIPv4Address()

Not sure what name, but something that makes it clearer the function is returning a boolean.